client/src/components/J40Map.tsx

```diff
@@ -173,7 +184,7 @@ const J40Map = () => {
         maxZoom={constants.GLOBAL_MAX_ZOOM}
         mapOptions={{hash: true}}
         width="100%"
-        height="52vw"
+        height={constants.isMobile && '90vw' || '44.8vw'}
```

Comment:
This might be causing the error of where it's not resizing on mobile. One idea is to create an entirely new instance of the ReactMapGL component and return if mobile is true.